pkgs/applications/editors/spacevim/init.toml

```diff
@@ -0,0 +1,61 @@
+# All SpaceVim options are below [options] snippet
+[options]
+    # set spacevim theme.
+    colorscheme = "gruvbox"
+    colorscheme_bg = 'dark'
+    enable_guicolors = true
+    statusline_separator = 'fire'
+    statusline_separator = 'fire'
+    buffer_index_type = 4
+    # Display file type icon on the tabline, If you do not have nerd fonts
+    # installed, please change the value to false
+    enable_tabline_filetype_icon = true
+    # Display current mode text on statusline, by default It is disabled,
+    # only color will be changed when switch modes.
+    enable_statusline_mode = true
+    timeoutlen = 500
+
+# SpaceVim default layer contains no plugins, but it has some better default config for vim and neovim.
+[[layers]]
+    name = 'default'
+
+[[layers]]
+    name = 'colorscheme'
+    enable = true
+
+[[layers]]
+    name = 'fzf'
+
+[[layers]]
+    name = 'shell'
+    default_position = 'top'
+    default_height = 30
+
+# This is an example for adding custom plugins lilydjwg/colorizer
+[[custom_plugins]]
+    name = 'lilydjwg/colorizer'
+    merged = false
+
+[[layers]]
+    name = 'edit'
+[[layers]]
+    name = 'VersionControl'
+[[layers]]
+    name = 'git'
+[[layers]]
+    name = 'autocomplete'
+    autocomplete_method = 'coc'
+    auto-completion-return-key-behavior = 'complete'
+    auto-completion-tab-key-behavior = 'cycle'
+[[layers]]
+    name = 'lang#ruby'
+[[layers]]
+    name = 'lang#nix'
+[[layers]]
+    name = 'lang#java'
+[[layers]]
+    name = 'lang#kotlin'
+[[layers]]
+    name = 'lang#sh'
+[[layers]]
+    name = 'lang#html'
```

Comment:
It is better to use unix fileformat. use `set ff=unix` in vim when edit this file.

Reply:
Fixed & rebased